Error: Triple:10074
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN-GB <http://data.southampton.ac.uk/point-of-service/42-sandwich.rdf> .):
u'<http://id.southampton.ac.uk/point-of-service/42-sandwich> <http://purl.org/openorg/ukfhrsRatingKey> "fhrs_5_en-GB"@EN-GB <http://data.southampton.ac.uk/point-of-service/42-sandwich.rdf> .'

Error: Triple:10699
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN <http://data.southampton.ac.uk/building/68-N9.rdf> .):
u'<http://id.southampton.ac.uk/> <http://www.w3.org/2000/01/rdf-schema#label> "University of Southampton"@EN <http://data.southampton.ac.uk/building/68-N9.rdf> .'

Error: Triple:23760
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN <http://data.southampton.ac.uk/building/75C.rdf> .):
u'<http://id.southampton.ac.uk/> <http://www.w3.org/2000/01/rdf-schema#label> "University of Southampton"@EN <http://data.southampton.ac.uk/building/75C.rdf> .'

Error: Triple:25218
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN <http://data.southampton.ac.uk/facility/F0066.rdf> .):
u'<http://id.southampton.ac.uk/> <http://www.w3.org/2000/01/rdf-schema#label> "University of Southampton"@EN <http://data.southampton.ac.uk/facility/F0066.rdf> .'

Error: Triple:30404
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN <http://data.southampton.ac.uk/org/F6.rdf> .):
u'<http://id.southampton.ac.uk/> <http://www.w3.org/2000/01/rdf-schema#label> "University of Southampton"@EN <http://data.southampton.ac.uk/org/F6.rdf> .'

Error: Triple:44463
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN <http://data.southampton.ac.uk/org/F4.rdf> .):
u'<http://id.southampton.ac.uk/> <http://www.w3.org/2000/01/rdf-schema#label> "University of Southampton"@EN <http://data.southampton.ac.uk/org/F4.rdf> .'

Error: Triple:95411
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN <http://data.southampton.ac.uk/org/BX.rdf> .):
u'<http://id.southampton.ac.uk/> <http://www.w3.org/2000/01/rdf-schema#label> "University of Southampton"@EN <http://data.southampton.ac.uk/org/BX.rdf> .'

Error: Triple:100054
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN <http://data.southampton.ac.uk/building/63F.rdf> .):
u'<http://id.southampton.ac.uk/> <http://www.w3.org/2000/01/rdf-schema#label> "University of Southampton"@EN <http://data.southampton.ac.uk/building/63F.rdf> .'

Error: Triple:100899
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN-GB <http://data.southampton.ac.uk/point-of-service/6-bar-food.rdf> .):
u'<http://id.southampton.ac.uk/point-of-service/6-bar-food> <http://purl.org/openorg/ukfhrsRatingKey> "fhrs_3_en-GB"@EN-GB <http://data.southampton.ac.uk/point-of-service/6-bar-food.rdf> .'

Error: Triple:101898
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN-GB <http://data.southampton.ac.uk/point-of-service/6-bar-food.rdf> .):
u'<http://id.southampton.ac.uk/point-of-service/6-bar-performance-nights> <http://purl.org/openorg/ukfhrsRatingKey> "fhrs_3_en-GB"@EN-GB <http://data.southampton.ac.uk/point-of-service/6-bar-food.rdf> .'

Error: Triple:102015
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN-GB <http://data.southampton.ac.uk/point-of-service/6-bar-food.rdf> .):
u'<http://id.southampton.ac.uk/point-of-service/6-bar> <http://purl.org/openorg/ukfhrsRatingKey> "fhrs_3_en-GB"@EN-GB <http://data.southampton.ac.uk/point-of-service/6-bar-food.rdf> .'

Error: Triple:102148
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN-GB <http://data.southampton.ac.uk/point-of-service/6-bar-food.rdf> .):
u'<http://id.southampton.ac.uk/point-of-service/6-bar-food-performance-nights> <http://purl.org/openorg/ukfhrsRatingKey> "fhrs_3_en-GB"@EN-GB <http://data.southampton.ac.uk/point-of-service/6-bar-food.rdf> .'

Error: Triple:110385
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN <http://data.southampton.ac.uk/building/36.rdf> .):
u'<http://id.southampton.ac.uk/> <http://www.w3.org/2000/01/rdf-schema#label> "University of Southampton"@EN <http://data.southampton.ac.uk/building/36.rdf> .'

Error: Triple:125267
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN <http://data.southampton.ac.uk/building/28.rdf> .):
u'<http://id.southampton.ac.uk/> <http://www.w3.org/2000/01/rdf-schema#label> "University of Southampton"@EN <http://data.southampton.ac.uk/building/28.rdf> .'

Error: Triple:133134
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN <http://data.southampton.ac.uk/building/5.rdf> .):
u'<http://id.southampton.ac.uk/> <http://www.w3.org/2000/01/rdf-schema#label> "University of Southampton"@EN <http://data.southampton.ac.uk/building/5.rdf> .'

Error: Triple:140199
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN-GB <http://data.southampton.ac.uk/vending-machine/29.rdf> .):
u'<http://id.southampton.ac.uk/point-of-service/85-lattes> <http://purl.org/openorg/ukfhrsRatingKey> "fhrs_5_en-GB"@EN-GB <http://data.southampton.ac.uk/vending-machine/29.rdf> .'

Error: Triple:160829
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN <http://data.southampton.ac.uk/building/63A.rdf> .):
u'<http://id.southampton.ac.uk/> <http://www.w3.org/2000/01/rdf-schema#label> "University of Southampton"@EN <http://data.southampton.ac.uk/building/63A.rdf> .'

Error: Triple:169704
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN <http://data.southampton.ac.uk/building/1730-11.rdf> .):
u'<http://id.southampton.ac.uk/> <http://www.w3.org/2000/01/rdf-schema#label> "University of Southampton"@EN <http://data.southampton.ac.uk/building/1730-11.rdf> .'

Error: Triple:193239
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN <http://data.southampton.ac.uk/org/FC.rdf> .):
u'<http://id.southampton.ac.uk/> <http://www.w3.org/2000/01/rdf-schema#label> "University of Southampton"@EN <http://data.southampton.ac.uk/org/FC.rdf> .'

Error: Triple:217646
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN <http://data.southampton.ac.uk/building/45.rdf> .):
u'<http://id.southampton.ac.uk/> <http://www.w3.org/2000/01/rdf-schema#label> "University of Southampton"@EN <http://data.southampton.ac.uk/building/45.rdf> .'

Error: Triple:231178
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN-GB <http://data.southampton.ac.uk/point-of-service/65-avenue.rdf> .):
u'<http://id.southampton.ac.uk/point-of-service/65-avenue> <http://purl.org/openorg/ukfhrsRatingKey> "fhrs_5_en-GB"@EN-GB <http://data.southampton.ac.uk/point-of-service/65-avenue.rdf> .'

Error: Triple:236745
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN <http://data.southampton.ac.uk/building/63G.rdf> .):
u'<http://id.southampton.ac.uk/> <http://www.w3.org/2000/01/rdf-schema#label> "University of Southampton"@EN <http://data.southampton.ac.uk/building/63G.rdf> .'

Error: Triple:253817
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN-GB <http://data.southampton.ac.uk/point-of-service/6-bar-food-performance-nights.rdf> .):
u'<http://id.southampton.ac.uk/point-of-service/6-bar-food-performance-nights> <http://purl.org/openorg/ukfhrsRatingKey> "fhrs_3_en-GB"@EN-GB <http://data.southampton.ac.uk/point-of-service/6-bar-food-performance-nights.rdf> .'

Error: Triple:254039
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN-GB <http://data.southampton.ac.uk/point-of-service/6-bar-food-performance-nights.rdf> .):
u'<http://id.southampton.ac.uk/point-of-service/6-bar-performance-nights> <http://purl.org/openorg/ukfhrsRatingKey> "fhrs_3_en-GB"@EN-GB <http://data.southampton.ac.uk/point-of-service/6-bar-food-performance-nights.rdf> .'

Error: Triple:254156
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN-GB <http://data.southampton.ac.uk/point-of-service/6-bar-food-performance-nights.rdf> .):
u'<http://id.southampton.ac.uk/point-of-service/6-bar> <http://purl.org/openorg/ukfhrsRatingKey> "fhrs_3_en-GB"@EN-GB <http://data.southampton.ac.uk/point-of-service/6-bar-food-performance-nights.rdf> .'

Error: Triple:254191
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN-GB <http://data.southampton.ac.uk/point-of-service/6-bar-food-performance-nights.rdf> .):
u'<http://id.southampton.ac.uk/point-of-service/6-bar-food> <http://purl.org/openorg/ukfhrsRatingKey> "fhrs_3_en-GB"@EN-GB <http://data.southampton.ac.uk/point-of-service/6-bar-food-performance-nights.rdf> .'

Error: Triple:255615
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN-GB <http://data.southampton.ac.uk/point-of-service/42-bridge-food.rdf> .):
u'<http://id.southampton.ac.uk/point-of-service/42-bridge-food> <http://purl.org/openorg/ukfhrsRatingKey> "fhrs_3_en-GB"@EN-GB <http://data.southampton.ac.uk/point-of-service/42-bridge-food.rdf> .'

Error: Triple:262072
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN <http://data.southampton.ac.uk/building/4.rdf> .):
u'<http://id.southampton.ac.uk/> <http://www.w3.org/2000/01/rdf-schema#label> "University of Southampton"@EN <http://data.southampton.ac.uk/building/4.rdf> .'

Error: Triple:263233
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN <http://data.southampton.ac.uk/facility/F0060.rdf> .):
u'<http://id.southampton.ac.uk/> <http://www.w3.org/2000/01/rdf-schema#label> "University of Southampton"@EN <http://data.southampton.ac.uk/facility/F0060.rdf> .'

Error: Triple:263661
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN <http://data.southampton.ac.uk/facility/F0062.rdf> .):
u'<http://id.southampton.ac.uk/> <http://www.w3.org/2000/01/rdf-schema#label> "University of Southampton"@EN <http://data.southampton.ac.uk/facility/F0062.rdf> .'

Error: Triple:264017
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN <http://data.southampton.ac.uk/org/F7.rdf> .):
u'<http://id.southampton.ac.uk/> <http://www.w3.org/2000/01/rdf-schema#label> "University of Southampton"@EN <http://data.southampton.ac.uk/org/F7.rdf> .'

Error: Triple:268359
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN-GB <http://data.southampton.ac.uk/point-of-service/42-cube-bar.rdf> .):
u'<http://id.southampton.ac.uk/point-of-service/42-cube-bar> <http://purl.org/openorg/ukfhrsRatingKey> "fhrs_5_en-GB"@EN-GB <http://data.southampton.ac.uk/point-of-service/42-cube-bar.rdf> .'

Error: Triple:271167
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN <http://data.southampton.ac.uk/building/1407.rdf> .):
u'<http://id.southampton.ac.uk/> <http://www.w3.org/2000/01/rdf-schema#label> "University of Southampton"@EN <http://data.southampton.ac.uk/building/1407.rdf> .'

Error: Triple:273974
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN <http://data.southampton.ac.uk/org/FH.rdf> .):
u'<http://id.southampton.ac.uk/> <http://www.w3.org/2000/01/rdf-schema#label> "University of Southampton"@EN <http://data.southampton.ac.uk/org/FH.rdf> .'

Error: Triple:275510
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN <http://data.southampton.ac.uk/building/75A.rdf> .):
u'<http://id.southampton.ac.uk/> <http://www.w3.org/2000/01/rdf-schema#label> "University of Southampton"@EN <http://data.southampton.ac.uk/building/75A.rdf> .'

Error: Triple:287518
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN <http://data.southampton.ac.uk/building/115.rdf> .):
u'<http://id.southampton.ac.uk/> <http://www.w3.org/2000/01/rdf-schema#label> "University of Southampton"@EN <http://data.southampton.ac.uk/building/115.rdf> .'

Error: Triple:290594
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN-GB <http://data.southampton.ac.uk/point-of-service/42-bridge-bar.rdf> .):
u'<http://id.southampton.ac.uk/point-of-service/42-bridge-bar> <http://purl.org/openorg/ukfhrsRatingKey> "fhrs_3_en-GB"@EN-GB <http://data.southampton.ac.uk/point-of-service/42-bridge-bar.rdf> .'

Error: Triple:291997
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN <http://data.southampton.ac.uk/building/54.rdf> .):
u'<http://id.southampton.ac.uk/> <http://www.w3.org/2000/01/rdf-schema#label> "University of Southampton"@EN <http://data.southampton.ac.uk/building/54.rdf> .'

Error: Triple:294489
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN-GB <http://data.southampton.ac.uk/point-of-service/42-cafe.rdf> .):
u'<http://id.southampton.ac.uk/point-of-service/42-cafe> <http://purl.org/openorg/ukfhrsRatingKey> "fhrs_4_en-GB"@EN-GB <http://data.southampton.ac.uk/point-of-service/42-cafe.rdf> .'

Error: Triple:298989
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN <http://data.southampton.ac.uk/building/27.rdf> .):
u'<http://id.southampton.ac.uk/> <http://www.w3.org/2000/01/rdf-schema#label> "University of Southampton"@EN <http://data.southampton.ac.uk/building/27.rdf> .'

Error: Triple:310748
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN-GB <http://data.southampton.ac.uk/generic-products-and-services/HotDrinks.rdf> .):
u'<http://id.southampton.ac.uk/point-of-service/6-bar-performance-nights> <http://purl.org/openorg/ukfhrsRatingKey> "fhrs_3_en-GB"@EN-GB <http://data.southampton.ac.uk/generic-products-and-services/HotDrinks.rdf> .'

Error: Triple:310795
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN-GB <http://data.southampton.ac.uk/generic-products-and-services/HotDrinks.rdf> .):
u'<http://id.southampton.ac.uk/point-of-service/72C-catering> <http://purl.org/openorg/ukfhrsRatingKey> "fhrs_5_en-GB"@EN-GB <http://data.southampton.ac.uk/generic-products-and-services/HotDrinks.rdf> .'

Error: Triple:310870
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN-GB <http://data.southampton.ac.uk/generic-products-and-services/HotDrinks.rdf> .):
u'<http://id.southampton.ac.uk/point-of-service/38-terrace> <http://purl.org/openorg/ukfhrsRatingKey> "fhrs_5_en-GB"@EN-GB <http://data.southampton.ac.uk/generic-products-and-services/HotDrinks.rdf> .'

Error: Triple:310949
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN-GB <http://data.southampton.ac.uk/generic-products-and-services/HotDrinks.rdf> .):
u'<http://id.southampton.ac.uk/point-of-service/68-cafe> <http://purl.org/openorg/ukfhrsRatingKey> "fhrs_5_en-GB"@EN-GB <http://data.southampton.ac.uk/generic-products-and-services/HotDrinks.rdf> .'

Error: Triple:311022
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN-GB <http://data.southampton.ac.uk/generic-products-and-services/HotDrinks.rdf> .):
u'<http://id.southampton.ac.uk/point-of-service/38-blue-room> <http://purl.org/openorg/ukfhrsRatingKey> "fhrs_5_en-GB"@EN-GB <http://data.southampton.ac.uk/generic-products-and-services/HotDrinks.rdf> .'

Error: Triple:311147
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN-GB <http://data.southampton.ac.uk/generic-products-and-services/HotDrinks.rdf> .):
u'<http://id.southampton.ac.uk/point-of-service/115-kiosk> <http://purl.org/openorg/ukfhrsRatingKey> "fhrs_5_en-GB"@EN-GB <http://data.southampton.ac.uk/generic-products-and-services/HotDrinks.rdf> .'

Error: Triple:311276
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN-GB <http://data.southampton.ac.uk/generic-products-and-services/HotDrinks.rdf> .):
u'<http://id.southampton.ac.uk/point-of-service/75A-catering> <http://purl.org/openorg/ukfhrsRatingKey> "fhrs_5_en-GB"@EN-GB <http://data.southampton.ac.uk/generic-products-and-services/HotDrinks.rdf> .'

Error: Triple:311351
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN-GB <http://data.southampton.ac.uk/generic-products-and-services/HotDrinks.rdf> .):
u'<http://id.southampton.ac.uk/point-of-service/6-bar> <http://purl.org/openorg/ukfhrsRatingKey> "fhrs_3_en-GB"@EN-GB <http://data.southampton.ac.uk/generic-products-and-services/HotDrinks.rdf> .'

Error: Triple:311448
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN-GB <http://data.southampton.ac.uk/generic-products-and-services/HotDrinks.rdf> .):
u'<http://id.southampton.ac.uk/point-of-service/42-piazza> <http://purl.org/openorg/ukfhrsRatingKey> "fhrs_5_en-GB"@EN-GB <http://data.southampton.ac.uk/generic-products-and-services/HotDrinks.rdf> .'

Error: Triple:311506
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN-GB <http://data.southampton.ac.uk/generic-products-and-services/HotDrinks.rdf> .):
u'<http://id.southampton.ac.uk/point-of-service/284a-broadlands-road> <http://purl.org/openorg/ukfhrsRatingKey> "fhrs_5_en-GB"@EN-GB <http://data.southampton.ac.uk/generic-products-and-services/HotDrinks.rdf> .'

Error: Triple:311520
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN-GB <http://data.southampton.ac.uk/generic-products-and-services/HotDrinks.rdf> .):
u'<http://id.southampton.ac.uk/point-of-service/38-arlott> <http://purl.org/openorg/ukfhrsRatingKey> "fhrs_5_en-GB"@EN-GB <http://data.southampton.ac.uk/generic-products-and-services/HotDrinks.rdf> .'

Error: Triple:311590
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN-GB <http://data.southampton.ac.uk/generic-products-and-services/HotDrinks.rdf> .):
u'<http://id.southampton.ac.uk/point-of-service/6-bar-food> <http://purl.org/openorg/ukfhrsRatingKey> "fhrs_3_en-GB"@EN-GB <http://data.southampton.ac.uk/generic-products-and-services/HotDrinks.rdf> .'

Error: Triple:311616
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN-GB <http://data.southampton.ac.uk/generic-products-and-services/HotDrinks.rdf> .):
u'<http://id.southampton.ac.uk/point-of-service/85-lattes> <http://purl.org/openorg/ukfhrsRatingKey> "fhrs_5_en-GB"@EN-GB <http://data.southampton.ac.uk/generic-products-and-services/HotDrinks.rdf> .'

Error: Triple:311696
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN-GB <http://data.southampton.ac.uk/generic-products-and-services/HotDrinks.rdf> .):
u'<http://id.southampton.ac.uk/point-of-service/63A-cafe> <http://purl.org/openorg/ukfhrsRatingKey> "fhrs_5_en-GB"@EN-GB <http://data.southampton.ac.uk/generic-products-and-services/HotDrinks.rdf> .'

Error: Triple:311770
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN-GB <http://data.southampton.ac.uk/generic-products-and-services/HotDrinks.rdf> .):
u'<http://id.southampton.ac.uk/point-of-service/the-crown-inn> <http://purl.org/openorg/ukfhrsRatingKey> "fhrs_2_en-GB"@EN-GB <http://data.southampton.ac.uk/generic-products-and-services/HotDrinks.rdf> .'

Error: Triple:311817
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN-GB <http://data.southampton.ac.uk/generic-products-and-services/HotDrinks.rdf> .):
u'<http://id.southampton.ac.uk/point-of-service/68-lattes> <http://purl.org/openorg/ukfhrsRatingKey> "fhrs_5_en-GB"@EN-GB <http://data.southampton.ac.uk/generic-products-and-services/HotDrinks.rdf> .'

Error: Triple:311890
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN-GB <http://data.southampton.ac.uk/generic-products-and-services/HotDrinks.rdf> .):
u'<http://id.southampton.ac.uk/point-of-service/163-university-road> <http://purl.org/openorg/ukfhrsRatingKey> "fhrs_3_en-GB"@EN-GB <http://data.southampton.ac.uk/generic-products-and-services/HotDrinks.rdf> .'

Error: Triple:311940
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN-GB <http://data.southampton.ac.uk/generic-products-and-services/HotDrinks.rdf> .):
u'<http://id.southampton.ac.uk/point-of-service/65-avenue> <http://purl.org/openorg/ukfhrsRatingKey> "fhrs_5_en-GB"@EN-GB <http://data.southampton.ac.uk/generic-products-and-services/HotDrinks.rdf> .'

Error: Triple:312101
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN-GB <http://data.southampton.ac.uk/generic-products-and-services/HotDrinks.rdf> .):
u'<http://id.southampton.ac.uk/point-of-service/6-bar-food-performance-nights> <http://purl.org/openorg/ukfhrsRatingKey> "fhrs_3_en-GB"@EN-GB <http://data.southampton.ac.uk/generic-products-and-services/HotDrinks.rdf> .'

Error: Triple:325416
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN-GB <http://data.southampton.ac.uk/point-of-service/68-cafe.rdf> .):
u'<http://id.southampton.ac.uk/point-of-service/68-cafe> <http://purl.org/openorg/ukfhrsRatingKey> "fhrs_5_en-GB"@EN-GB <http://data.southampton.ac.uk/point-of-service/68-cafe.rdf> .'

Error: Triple:326416
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN-GB <http://data.southampton.ac.uk/point-of-service/68-cafe.rdf> .):
u'<http://id.southampton.ac.uk/point-of-service/68-lattes> <http://purl.org/openorg/ukfhrsRatingKey> "fhrs_5_en-GB"@EN-GB <http://data.southampton.ac.uk/point-of-service/68-cafe.rdf> .'

Error: Triple:327209
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN <http://data.southampton.ac.uk/facility/F0020.rdf> .):
u'<http://id.southampton.ac.uk/> <http://www.w3.org/2000/01/rdf-schema#label> "University of Southampton"@EN <http://data.southampton.ac.uk/facility/F0020.rdf> .'

Error: Triple:330351
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN-GB <http://data.southampton.ac.uk/point-of-service/40-stags.rdf> .):
u'<http://id.southampton.ac.uk/point-of-service/40-stags> <http://purl.org/openorg/ukfhrsRatingKey> "fhrs_4_en-GB"@EN-GB <http://data.southampton.ac.uk/point-of-service/40-stags.rdf> .'

Error: Triple:337109
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN-GB <http://data.southampton.ac.uk/point-of-service/63A-cafe.rdf> .):
u'<http://id.southampton.ac.uk/point-of-service/63A-cafe> <http://purl.org/openorg/ukfhrsRatingKey> "fhrs_5_en-GB"@EN-GB <http://data.southampton.ac.uk/point-of-service/63A-cafe.rdf> .'

Error: Triple:342334
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN-GB <http://data.southampton.ac.uk/point-of-service/38-fotg.rdf> .):
u'<http://id.southampton.ac.uk/point-of-service/38-terrace> <http://purl.org/openorg/ukfhrsRatingKey> "fhrs_5_en-GB"@EN-GB <http://data.southampton.ac.uk/point-of-service/38-fotg.rdf> .'

Error: Triple:342406
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN-GB <http://data.southampton.ac.uk/point-of-service/38-fotg.rdf> .):
u'<http://id.southampton.ac.uk/point-of-service/38-blue-room> <http://purl.org/openorg/ukfhrsRatingKey> "fhrs_5_en-GB"@EN-GB <http://data.southampton.ac.uk/point-of-service/38-fotg.rdf> .'

Error: Triple:342577
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN-GB <http://data.southampton.ac.uk/point-of-service/38-fotg.rdf> .):
u'<http://id.southampton.ac.uk/point-of-service/38-arlott> <http://purl.org/openorg/ukfhrsRatingKey> "fhrs_5_en-GB"@EN-GB <http://data.southampton.ac.uk/point-of-service/38-fotg.rdf> .'

Error: Triple:351663
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN <http://data.southampton.ac.uk/org/F2.rdf> .):
u'<http://id.southampton.ac.uk/> <http://www.w3.org/2000/01/rdf-schema#label> "University of Southampton"@EN <http://data.southampton.ac.uk/org/F2.rdf> .'

Error: Triple:354835
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN-GB <http://data.southampton.ac.uk/point-of-service/68-lattes.rdf> .):
u'<http://id.southampton.ac.uk/point-of-service/68-lattes> <http://purl.org/openorg/ukfhrsRatingKey> "fhrs_5_en-GB"@EN-GB <http://data.southampton.ac.uk/point-of-service/68-lattes.rdf> .'

Error: Triple:355444
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN-GB <http://data.southampton.ac.uk/point-of-service/68-lattes.rdf> .):
u'<http://id.southampton.ac.uk/point-of-service/68-cafe> <http://purl.org/openorg/ukfhrsRatingKey> "fhrs_5_en-GB"@EN-GB <http://data.southampton.ac.uk/point-of-service/68-lattes.rdf> .'

Error: Triple:356681
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN <http://data.southampton.ac.uk/facility/F0061.rdf> .):
u'<http://id.southampton.ac.uk/> <http://www.w3.org/2000/01/rdf-schema#label> "University of Southampton"@EN <http://data.southampton.ac.uk/facility/F0061.rdf> .'

Error: Triple:363163
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN-GB <http://data.southampton.ac.uk/point-of-service/75A-catering.rdf> .):
u'<http://id.southampton.ac.uk/point-of-service/75A-catering> <http://purl.org/openorg/ukfhrsRatingKey> "fhrs_5_en-GB"@EN-GB <http://data.southampton.ac.uk/point-of-service/75A-catering.rdf> .'

Error: Triple:377112
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN <http://data.southampton.ac.uk/facility/F0065.rdf> .):
u'<http://id.southampton.ac.uk/> <http://www.w3.org/2000/01/rdf-schema#label> "University of Southampton"@EN <http://data.southampton.ac.uk/facility/F0065.rdf> .'

Error: Triple:381894
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN-GB <http://data.southampton.ac.uk/point-of-service/6-bar.rdf> .):
u'<http://id.southampton.ac.uk/point-of-service/6-bar> <http://purl.org/openorg/ukfhrsRatingKey> "fhrs_3_en-GB"@EN-GB <http://data.southampton.ac.uk/point-of-service/6-bar.rdf> .'

Error: Triple:382096
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN-GB <http://data.southampton.ac.uk/point-of-service/6-bar.rdf> .):
u'<http://id.southampton.ac.uk/point-of-service/6-bar-performance-nights> <http://purl.org/openorg/ukfhrsRatingKey> "fhrs_3_en-GB"@EN-GB <http://data.southampton.ac.uk/point-of-service/6-bar.rdf> .'

Error: Triple:382222
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN-GB <http://data.southampton.ac.uk/point-of-service/6-bar.rdf> .):
u'<http://id.southampton.ac.uk/point-of-service/6-bar-food> <http://purl.org/openorg/ukfhrsRatingKey> "fhrs_3_en-GB"@EN-GB <http://data.southampton.ac.uk/point-of-service/6-bar.rdf> .'

Error: Triple:382333
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN-GB <http://data.southampton.ac.uk/point-of-service/6-bar.rdf> .):
u'<http://id.southampton.ac.uk/point-of-service/6-bar-food-performance-nights> <http://purl.org/openorg/ukfhrsRatingKey> "fhrs_3_en-GB"@EN-GB <http://data.southampton.ac.uk/point-of-service/6-bar.rdf> .'

